doc/api/dns.md

```diff
@@ -792,6 +792,10 @@ priority than [`--dns-result-order`][]. When using [worker threads][],
 [`dns.setDefaultResultOrder()`][] from the main thread won't affect the default
 dns orders in workers.
 
+## `dns.getDefaultResultOrder()`
+
+Get the value of `dnsOrder`.
```

Comment:
I think we would need to link to where this is defined.

Reply:
You mean to link `dnsOrder` to here?
https://github.com/nodejs/node/blob/db81af61ce0d55c052723e30ac7f6b7ad60f70c3/lib/internal/dns/utils.js#L200

Reply:
The definition in the documentation, not in the code. Something like
```suggestion
Get the default value for `verbatim` in [`dns.lookup()`][] and
[`dnsPromises.lookup()`][]. The value could be:

* `ipv4first`: for `verbatim` defaulting to `false`.
* `verbatim`: for `verbatim` defaulting to `true`.
```